asyncscope.md

```diff
@@ -1825,7 +1825,32 @@ spec here
 
 ## `execution::nest()`
 
-spec here
+::: add
+__§34.9.11.14 `std::execution::nest` [exec.nest]__
+
+[1]{.pnum} `nest` nests its input sender within the dynamic lifetime of the async scope referenced by its input async
+scope token.
+
+[2]{.pnum} The name `nest` denotes a customization point object. For subexpressions `sndr` and `token`, let `Sndr` be
```

Comment:
I wonder if this should explicitly be denoted as a Mandates here per https://eel.is/c++draft/description

Reply:
I don't know. I modeled this section after paragraph 2 of [section 34.9.11.6 of P2300](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2024/p2300r9.html#spec-execution.senders.adaptor.then), which describes `then`, `upon_error`, and `upon_stopped` as customization point objects that are ill-formed under certain conditions.

Reply:
This is the way.